src/Interactive/CsiCore/project.json

```diff
@@ -54,7 +54,6 @@
     "System.Xml.XPath.XDocument": "4.3.0"
   },
   "frameworks": {
-    // We don't actually target netstandard1.6; this is to work around https://github.com/dotnet/roslyn/issues/12458
```

Comment:
@jasonmalinowski Nope. In fact, it removes errors. This construction is illegal in the 4.0 version of NuGet used by VS.